java/pom.xml

```diff
@@ -27,18 +27,17 @@
     </distributionManagement>
   
     <dependencies>
-        <dependency>
+        <!--<dependency>
             <groupId>commons-beanutils</groupId>

```

Comment:
Can we consider maybe moving away from checking in these new comments since we can also perform a compare inside our IDE's to see what was changed?